docs/cli/add.md

```diff
@@ -0,0 +1,16 @@
+# Add Block
+
+The `add` command will add new blocks to your nio project. These blocks are pulled from the official [nio-blocks GitHub](https://github.com/nio-blocks) repository. All of the requirements for the block will be installed along with it.
```

Comment:
take out the 'will' and just use present tense throughout: the `add` command adds new blocks to your nio project.